sdk/ml/azure-ai-ml/tests/workspace/unittests/test_workspace_operations.py

```diff
@@ -5,7 +5,7 @@
 from pytest_mock import MockFixture
 
 from azure.ai.ml import MLClient, load_workspace
-from azure.ai.ml._restclient.v2022_12_01_preview.models import (
+from azure.ai.ml._restclient.v2023_02_01_preview.models import (
```

Comment:
According to previous comment, can workspace continue to use 2022_12_01_preview version?